use-timescale/services/connection-pooling.md

```diff
@@ -0,0 +1,121 @@
+IoT applications often use an event-based architecture where data is read or
+written from the database for very short amount of time.
+
+<EarlyAccess />
+
+Your application rapidly opens and closes connections while the pooler
+maintains a set of long-running connections to the database. This improves
+performance because the pooler open the connections in advance,  allowing the
+application to open many short-lived connections, while the database opens few,
+long-lived connections.
+
+## Pool types
+
+When you create a connection pooler, there are two pool types to choose from:
+session or transaction. Each pool type uses a different mode to handle
+connections.
+
+Session pools allocate a connection from the pool until they are closed by the
+application, similar to a regular PostgreSQL connection. When the application
+closes the connection, it is sent back to the pool.
+
+Transaction pool connections are allocated only for the duration of the
+transaction, releasing the connection back to the pool when the transaction
+ends. If your application opens and closes connections frequently, choose the
+transaction pool type.
+
+## Connection allocations
+
+A connection pooler manages connections to both the database itself, and the
+client application.
+
+The number of database connections allocated in each pool is managed by the
+`max_connections` parameter set in the service. The pooler keeps allocating
+connections until 60% of the total `max_connections`. 45% of the
+`max_connections` parameter is used for the session pool, and the remaining 15%
+is available for the transaction pool.
+
+The connection pooler can then allocate connections for the client application.
+Up to 45% of `max_connections` is available for the session pool, and up to 100%
+of `max_connections` is available for the transaction pool.
+
+For example, if `max_connections` is set to 500, the maximum number of database
+connections is 300, or 60% of `max_connections`. For the client application, in
+session mode, the maximum number of connections is 225, which limits the total
+number of database connections to 225. In transaction mode, the maximum number
+of connections is 500, which limits the total number of database connections to
+75.
+
+## Add a connection pooler
+
+When you create a new service, you can also create a connection
+pooler. Alternatively, you can add a connection pooler to an existing service in
+the Timescale portal.
+
+<Procedure>
+
+### Adding a connection pooler
+
+1.  [Log in to the Timescale portal][cloud-login] and click the service
+    you want to add a connection pooler to.
+1.  In the `Connection info` section, navigate to the `Connection pooler` tab,
+    and click `Add connection pooler`.
+1.  When the pooler has been added, your pooler connection details are displayed
+    in the `Connection pooler` tab. Use this information to connect to your
+    pooler.
+1.  By default, your pooler is started as a session pool. You can change this to
```

Comment:
This is incorrect: the pooler supports both modes simultaneously, but the connection configuration is different. A drop-down menu shows the correct connection string for the corresponding mode (session or transaction). Consider adding a few details: the pooler connection string is almost identical to the regular connection string, but uses a different port (shared between the transaction and the session mode). The session mode connection string differs from the transaction mode one by the database name. Example:

If a regular (direct) connection string for a service is:
```
postgres://user:password@service.example.cloud.timescale.com:30133/tsdb?sslmode=require
```
the session pooler connection will use a different port:
```
postgres://user:password@service.example.cloud.timescale.com:29303/tsdb?sslmode=require
```

and the transaction pooler will use the same port *as the session pooler*, but the database name will be `tsdb_transaction` instead of `tsdb`:

```
postgres://user:password@service.example.cloud.timescale.com:29303/tsdb_transaction?sslmode=require
```

In both the transaction and the session modes, all parts of the service URL before the port (the user, the password, the host) stay the same.

```

Reply:
Added this explanation under Pool Types ☝️